geometry_msgs/Pose pt1
geometry_msgs/Pose pt2
